frontend/smart-kickers-game/src/App.js

```diff
@@ -1,17 +1,29 @@
 import { useEffect, useState } from 'react';
 import './App.css';
-import { resetGame } from './apis/Game';
-import { Button } from './components/Button';
-import GameResults from './components/GameResults.js';
+import { Button } from './components/Button/Button';
+import GameResults from './components/Game/GameResults.js';
+import { resetGame } from './apis/resetGame';
+import GameStatistics from './components/Game/GameStatistics.js';
+import { library } from '@fortawesome/fontawesome-svg-core';
+import { faPerson } from '@fortawesome/free-solid-svg-icons';
 
 import config from './config';
 
+library.add(faPerson);
+
 function App() {
   const [blueScore, setBlueScore] = useState(0);
   const [whiteScore, setWhiteScore] = useState(0);
+  const [toggleGameScore, setToggleGameScore] = useState(false);
```

Comment:
Please consider changing the name of that variable to something that describes that the statistics are displayed or not, it is the boolean value, so the name should be adjusted to that.